CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## [0.24.1](https://github.com/awslabs/aws-cdk/compare/v0.24.0...v0.24.1) (2019-02-07)
```

Comment:
⚠️ BAD HEADING

Reply:
Now I get what the problem is: the first heading is ignored by the changelog parser, it's not that it's H2

Reply:
Yep